### ISSUE TITLE
pff.parametrize a class Test

### ISSUE THREAD

**Issue by UmeSiyah** [kalekundert/parametrize_from_file#21]
Hello,

I'm trying to parametrize a class Test with `@pff.parametrize`, but it does nothing and don't raise any error. My tests that worked well with `@pff.parametrize` on each method of `TestEpisode`, are not collected.

The Python file
```python
@pff.parametrize
class TestEpisode:
        def test_get_epi(self, input_episode: dict, excepted_episode: dict)
```

The Yaml
```yaml
TestEpisode:
  - input_episode:
      *episode_01
    input_sequence:
      *sequence_01
    excepted_episode: *episode_01
    excepted_sequence: *sequence_01
```

Am I missing something ?

Thanks you

**Comment by kalekundert** [comment]
Thanks for the report!  You're not missing anything; this is a bug.  I haven't totally figured it out yet, but I think the basic issue is that the method I'm using to make decorators only works for functions, not classes.  Hopefully I'll be able to get a fix out quickly.

**Comment by kalekundert** [comment]
Ok, this should be fixed now.  I also uploaded a new version with the fix (0.19.0) to PyPI.  Let me know if you have any other problems!
